qutebrowser/completion/completionwidget.py

```diff
@@ -262,48 +261,45 @@ def completion_item_focus(self, which):
         elif config.get('completion', 'show') == 'auto':
```

Comment:
Oh, I just noticed there's actually a Qt warning when trying to press `tab` and it doesn't work here:

```
Can't select indexes from different model or with different parents
```

Stack:

```
  [...]
  File ".../qutebrowser/commands/command.py", line 525, in run
    self.handler(*posargs, **kwargs)
  File ".../qutebrowser/completion/completionwidget.py", line 254, in completion_item_focus
    idx, QItemSelectionModel.ClearAndSelect | QItemSelectionModel.Rows)
  File ".../qutebrowser/utils/log.py", line 443, in qt_message_handler
    stack = ''.join(traceback.format_stack())
```